star_tides/requirements.txt

```diff
@@ -6,4 +6,5 @@ google-auth==1.30.0
 requests==2.25.1
 pytest==6.2.4
 pylint==2.8.2
+astroid==2.5.7
```

Comment:
What is this for?

Reply:
It's required by pylint. We want it fixed because it's the version associated to this version of pylint.